features/steps.rb

```diff
@@ -281,3 +281,10 @@
     File.open(file, "w") { |f| f.write(contents) }
   end
 end
+
+Given(/^the file "([^"]+)" content is:$/) do |file, content|
```

Comment:
There's no need: you can simply re-use the `Given a file named "..." with:` step provided by Aruba.